src/main/java/io/aiven/kafka/connect/common/output/jsonwriter/ObjectMapperProvider.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2023 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.common.output.jsonwriter;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+
+/**
+ * Wraps an object mapper for reuse. There should be only a single object mapper created for the whole application.
+ */
+public class ObjectMapperProvider {
+
+    private static final ObjectMapper OBJECT_MAPPER;
+
+    static {
+        OBJECT_MAPPER = new ObjectMapper();
+        OBJECT_MAPPER.setNodeFactory(JsonNodeFactory.withExactBigDecimals(true));
+    }
+
+    public static ObjectMapper get() {
+        return OBJECT_MAPPER;
+    }
```

Comment:
could it happen that after 
```java
 OBJECT_MAPPER = new ObjectMapper();
 ```
 there could be a parallel call to `get()` which could return `OBJECT_MAPPER` without required node factory set?

Reply:
You mean two distinct object mappers? Like this:
```suggestion
    private static final ObjectMapper OBJECT_MAPPER_EXACT_DECIMALS;
    private static final ObjectMapper OBJECT_MAPPER;

    static {
        OBJECT_MAPPER = new ObjectMapper();
        OBJECT_MAPPER_EXACT_DECIMALS = new ObjectMapper();
        OBJECT_MAPPER_EXACT_DECIMALS.setNodeFactory(JsonNodeFactory.withExactBigDecimals(true));
    }

    public static ObjectMapper get() {
        return OBJECT_MAPPER;
    }
    
    public static ObjectMapper getWithExactDecimals() {
        return OBJECT_MAPPER_EXACT_DECIMALS;
    }
```

Reply:
no, I meaan that probably need to have `synchronized` for static block to prevent cases when `get` method could return unpredictable value

Reply:
Sure, I connected the comment to what @jeqo's mental model. I'll investigate a bit on this.

Reply:
@snuyanzin The static initializer is thread-safe: https://docs.oracle.com/javase/specs/jls/se11/html/jls-12.html#jls-12.4.2

Reply:
yep you're right, thanks